imagetest/test_suites/oslogin/ssh_test.go

```diff
@@ -42,11 +44,14 @@ const (
 	normal2FAKey = "normal-2fa-key"
 	admin2FAKey  = "admin-2fa-key"
 
-	// keys
-	normalUserSshKey = "normal-user-ssh-key"
-	adminUserSshKey  = "admin-user-ssh-key"
-	normal2FASshKey  = "normal-2fa-ssh-key"
-	admin2FASshKey   = "admin-2fa-ssh-key"
+	// SSH keys
+	normalUserSSHKey = "normal-user-ssh-key"
+	adminUserSSHKey  = "admin-user-ssh-key"
+	normal2FASSHKey  = "normal-2fa-ssh-key"
+	admin2FASSHKey   = "admin-2fa-ssh-key"
+
+	// Time to wait for agent check
+	waitAgent = time.Second * 15
```

Comment:
If we are setting a const for this can we set the other sleep times as well? at least the ones your are changing. Please add context to why the values seem to be the right one. I general a fall back retry strategy seem always to be a better option rather than these arbitrary sleep but I'm not blocking this PR because of that, let's see if we can address it in a specific project later.

Reply:
Acknowledged.

I'll add context here for now, but I'll add it in comments as well in a followup commit.
The time for waitAgent is, obviously, for the TestAgent check. The TestAgent itself has a few `sleep` statements as it needs to wait for metadata changes to propagate and for the guest agent to respond to the metadata changes accordingly. So this time is determined by the total amount of sleep time (8 seconds at the time), the runtime of the agent check test (around 1-2.5s), with a little bit of extra padding to make extra sure things are ready.